Lectures/Lecture1_theOpenScienceEthos.md

```diff
@@ -10,27 +10,28 @@
 ## Development goal
 To gather enough knowledge about the recent history of the open science movement in order to appreciate the necessity of taking an informed position and taking the necessary steps towards mastering the needed skills.
 
-![5SchoolsOfOpenScience](../Week1/SchoolsofOpenscience.jpg?raw=true)
+![5SchoolsOfOpenScience](../Resources/SchoolsofOpenscience.jpg?raw=true)
 
 ## Motivation: Why open science, how did we get here? 
-Adopting and implementing open science practices in academia has entered mainstream science policy discussions for at least a decade. Despite several national and international efforst, there are still several persistent misrepresentations of the basic open science needs and achievements. 
-In this lecture, student are introduced to various aspects of open science and how people and organizations are getting there from the current state of science. 
+Adopting and implementing open science practices in academia has entered mainstream science policy discussions for at least a decade. Despite several national and international efforts, there are still several persistent misrepresentations of the open science basic demands and main achievements. 
+In this lecture, student are introduced to a brief history of open science and how people and organizations are dealing with this global transformation of science. 
 After the introducing different schools of open science students are encouraged to take a point of view on the relation of each classification to their potential future projects and to name consequences of open science for their careers, and for science in general. 
-Besides that, the international context and access to scientific knowledge will be discussed, as well as the opposition to the inacted new research policies in connection with open science and the new system of recognition in academia. 
+Besides that, the international context and especially fair access to scientific knowledge will be discussed, as well as the opposition to the inacted new research policies in connection with the open science movement and the new system of recognition in academia. 
 
 ## Vision of outcome
 + Course participants are able to engage in an informed discussion about the requirements and achievements of the open science movement
```

Comment:
Why not call them 'students' throughout? Why have you chosen the (more formal) 'course participants'? And if you prefer that, should we also change to that term in other lectures?

Reply:
I sometimes think that the course could be useful beyond the UU, hence the use of participants. I agree that we should be consistent. Will do an overall search and replace before moving it to gitlab.